doc/tutorials/using_serial_port.rst

```diff
@@ -1,35 +1,38 @@
-.. _connect_serial_port:
+.. _enabling_serial_port:
```

Comment:
The label you change here is used in another tutorial (`using_hybrid_mode_on_nuc.rst`).

I tried a `make doc` on  your PR but it still throws some errors. Have you validated that it works on your side?

Reply:
Done!